data/heliarch intro.txt

```diff
@@ -0,0 +1,1570 @@
+			`The only reason the Heliarchs must have asked you to land on Humanika is to see how the human inhabitants interact with the Quarg-controlled robots here. You land as asked, and can't seem to notice anything different with your ship.`
+				decline
+
+
+
+mission "Heliarch Recon F"
+	invisible
+	landing
+	source "Forpelog"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`	While you told them this was nothing more than a barren mining outpost, the Heliarchs still insisted on gathering intel on this small moon. Regardless of it's usefulness, you land as they asked, and let your scanners get a proper reading on this world.`
+				decline
+
+
+
+mission "Heliarch Recon G"
+	invisible
+	landing
+	source "Grakhord"
+	destination "Earth"
+	to offer
+		has "Heliarch Recon 1: active"
+	on offer
+		conversation
+			`As instructed, you land on Grakhord and let your ship work normally while the Heliarch sensor does its thing. Though they are very advanced, you doubt whether the modified scanner could penetrate the layers of rock to fully reveal the Quarg mining operation.`
+				decline
+
+
+
+event "heliarch recon break"
+
+
+
+mission "Heliarch Recon 2-A"
+	name "Scanning Preparations"
+	description "Install an Outfit Scanner, then land on <destination>, where the Heliarch will modify it to be able to scan Quarg ships the way they need it."
+	landing
+	source
+		government "Heliarch"
+		not planet "Ring of Wisdom"
+	destination "Ring of Wisdom"
+	to offer
+		has "event: heliarch recon break"
+		not "Resistance: Questions: offered"
+	on offer
+		conversation
+			`Some Heliarchs approach you as you land. You think you can recognize the Saryd of the group as one of those who asked you to land on all the Quarg-owned worlds some time ago.`
+			`	"The honorable Captain, we once again are pleased to greet," he says.`
+			`	"Another fortuitous meeting, we hope this will be."`
+			`	"In mind, a new strategy we have. Your assistance, we once again require."`
+			choice
+				`	"What do you need me to do?"`
+				`	"I'm sorry, but I'm not willing to risk my ship any longer to help you spy on the Quarg."`
+					decline
+			`	"If too complex and protected, the Quarg's rings are, perhaps something new, from their ships we might learn."`
+			`	"Scan them during our war, we of course did, but much wiser and more sofisticated, we and our devices have become."`
+			`	"To use one of our own scanners, we cannot ask of thee. Much risk, it would pose. Alas, if to <destination>, a human Outfit Scanner you bring, alter it, we could."`
+			`	You agree to head there once you have equipped an Outfit Scanner, and they thank you for your willingness to help.`
+				accept
+	on complete
+		require "Outfit Scanner"
+
+
+
+mission "Heliarch Recon 2-B"
+	name "Silent Scan"
+	description "Now that the Heliarchs have altered your Outfit Scanner, head to <waypoint> and scan some Quarg ships there, then report back to <destination>."
```

Comment:
I tried playing this branch, and the `<waypoint>` wasn't replaced in the pop-up text.  The pop-up text said:
```
Now that the Heliarchs have altered your Outfit Scanner, 
head to <waypoint> and scan some Quarg ships there, 
then report back to the Ring of Wisdom in the Ekuarik
system.
```
I'm not sure why it didn't get replaced, since `waypoint` is defined below.  The Enif system was circled so I was still able to figure out where to go.

Reply:
yeah, should be <waypoints>. I'll most likely be fixing that and some other stuff I myself found in the next few weeks